command/daemon/command.go

```diff
@@ -58,7 +58,7 @@ func New(config Config) (Command, error) {
 
 	newCommand.cobraCommand.PersistentFlags().StringSlice(f.Config.Dirs, []string{"."}, "List of config file directories.")
 	newCommand.cobraCommand.PersistentFlags().StringSlice(f.Config.Files, []string{"config"}, "List of the config file names. All viper supported extensions can be used.")
-
+	newCommand.cobraCommand.PersistentFlags().Bool(f.Server.Enable.Debug.Server, false, "Enable debug server at http://127.0.0.1:6060/debug.")
```

Comment:
Why not on by default?

Reply:
If someone else uses this library, it would be a nasty default.

Reply:
Makes sense. Thanks!